whack-a-mole/script.js

```diff
@@ -0,0 +1,159 @@
+(function startFunction(){
+const holes = document.querySelectorAll('.hole');
+const moles = document.querySelectorAll('.mole');
+const scoreScreen = document.getElementById('score');
+const startButton = document.getElementById('start-button');
+const lavelChange = document.getElementById('level-change');
+const minMax = document.getElementById('first-level');
+const gameContinue = document.querySelector('.game-continue');
+const buttons = document.querySelectorAll('.button');
```

Comment:
I know that there are only two buttons in your game, but better to have more specific variable and selector name. 
Now it is hard to understand, what are this buttons fore. 
```suggestion
const continueGameButtons = document.querySelectorAll('.continue-game-modal .button');
```